sunpy/util/table_attribute.py

```diff
@@ -0,0 +1,119 @@
+"""
+This file backports the TableAttribute functionality from astropy 4.1 it can be
+removed when we depend on astropy 4.1.
```

Comment:
This should all be private (probably _table_attribute.py) if it's not intended to be user facing.